image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:bionic-20180426
```

Comment:
Any reason you've used this tag format instead of the version number?

Reply:
When I submitted the PR, the 18.04 tag was still pointing to a beta. They're in sync now, so it can be changed to 18.04